packaging/package.sh

```diff
@@ -19,17 +19,19 @@ function sign() {
   codesign --deep --sign "${CODESIGN_IDENTITY}" --options runtime --force ${opts} "$1"
 }
 
-version=$(ls "${BASEDIR}/root/Library/CodeReady Containers/")
+binDir="${BASEDIR}/root/Applications/CodeReady Containers.app/Contents/Resources"
+
+version=$("${binDir}/crc" version -o json | jq -r .version)
```

Comment:
We are adding `jq` as requirement which means it should be present on the RCM signing box also?

Reply:
Yes thanks, I removed this dependency. I added a small VERSION file in the tarball we give to RCM. The script just has to read it.